src/reflect/scala/reflect/internal/tpe/TypeComparers.scala

```diff
@@ -30,6 +30,7 @@ trait TypeComparers {
     override def toString = tp1+" <:<? "+tp2
   }
 
+  // With Threadlocal it fails with NPE in GenBCode.scala:94
```

Comment:
Interesting, because in `SynchronizedTypes` it is `ThereadLocal`. We will need to investigate deeper.

Reply:
agree